crates/sp-domains-fraud-proof/src/fraud_proof.rs

```diff
@@ -37,7 +37,7 @@ impl ExecutionPhase {
             // TODO: Replace `DomainCoreApi_initialize_block_with_post_state_root` with `Core_initalize_block`
             // Should be a same issue with https://github.com/paritytech/substrate/pull/10922#issuecomment-1068997467
             Self::InitializeBlock => "DomainCoreApi_initialize_block_with_post_state_root",
-            Self::ApplyExtrinsic { .. } => "BlockBuilder_apply_extrinsic",
+            Self::ApplyExtrinsic { .. } => "DomainCoreApi_apply_extrinsic_with_post_state_root",
```

Comment:
With this change the `proving_method` and `verifying_method` are identical, we can merge them into one.